vertical_ngo/models/res_config.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class ngo_config_settings(models.TransientModel):
+    _name = 'ngo.config.settings'
+    _inherit = 'res.config.settings'
+
+    module_framework_agreement_requisition = fields.Boolean(
+        'Create tenders to negociate a framework agreements',
+        help="To allow your NGO to manage tenders when negociating "
+        "framework agreements from possible suppliers."
+    )
+    module_framework_agreement_sourcing = fields.Boolean(
+        'Allow to source a logistics requisition with a framework agreement',
+        help="if your NGO has framework agreements with suppliers "
+        "which you use to source logistics requisitions."
+    )
+    module_logistic_budget = fields.Boolean(
```

Comment:
Your module name(s) is 'logistic_budget'. It should be 'logistics_budget' ?
May be you could use https://github.com/bealdav/erp_rename/blob/master/erp_rename.py
I'm sure you can do a nice improvement and embedds in your community maintener tools